docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,21 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server can be used to collect cloud-native metrics from Couchbase Servers.
```

Comment:
We probably need to be more explicit that this is Prometheus in CMOS to distinguish the Prometheus instance in Couchbase Server 7+

Reply:
> We probably need to be more explicit that this is Prometheus in CMOS to distinguish the Prometheus instance in Couchbase Server 7+

From prometheus instances do you mean exporter here?

Reply:
I would try to keep abstract minimal. Following sections explain things in more clarity.

Reply:
No, Couchbase Server 7 provides its own Prometheus server internally which we can scrape - we do not need the exporter and the intention is to focus on this for 0.1.

Reply:
So my concern is that the "Couchbase Prometheus Server" is ambiguous - do you mean the one in server or the one in CMOS?

Reply:
Updated info about both